README.md

```diff
@@ -26,6 +26,10 @@ The [bloomfilter](https://en.wikipedia.org/wiki/Bloom_filter) based PSI: an *ins
 
 A complete test suite for all PSIs is present [here](test/psi). Don't hesitate to take a look and help us improve the quality of the testing by reporting problems and observations!
 
+# benchmarks
+
+A benchmark on the running times of different PSI protocols of this library is provided [here](benchmark/README.md).
```

Comment:
See runtime benchmarks of the different PSI protocols [here](benchmark/README.md).